### ISSUE TITLE
Custom scrollbars render funny

### ISSUE THREAD

**Issue by dmt0** [handsontable/handsontable#5576]
### Description
When I use custom scrollbars, `ht_clone_top` and `ht_clone_left` elements assume that my scrollbar is of the default size, so it renders funny:
![image](https://user-images.githubusercontent.com/2717746/48294951-c861b600-e455-11e8-9e9b-1248032ab148.png)
Notice the white rectangles in the rows and columns.

### Your environment
* Handsontable version: 6.1.0
* Browser Name and version: Chrome 70
* Operating System: MacOS

**Comment by AMBudnik** [comment]
Welcome @dmt0 

have you tried to add it to the container that holds the instance? I do not know which scrollbar it is (container, parent)?

**Comment by dmt0** [comment]
Yes, I tried adding it to the container, and tried a couple of layers up, and a few layers within - `hot_container`, `ht_master`, `wtHolder`, etc. Nothing works except adding it globally.

Here's a fiddle from a similar question here: http://jsfiddle.net/sab6erkq/
You can try doing it there, and see the funny display issues reappear.

**Comment by AMBudnik** [comment]
This Fiddle seems to work fine. I do not see an empty space.

![image](https://user-images.githubusercontent.com/10757813/48702816-2fb7fc80-ebf2-11e8-948e-51e40bbf3a13.png)

I have tested it on Chrome 69/MacOS and Chrome 70/Windows 10

**Comment by AMBudnik** [comment]
Sorry, but I haven't found any workaround for it.
Here's a related issue https://github.com/handsontable/handsontable/issues/272

**Comment by dmt0** [comment]
> Scroll bar styling is now possible in version 0.8.0

So the "feature" already exists? So that makes it a bug? :)

**Comment by AMBudnik** [comment]
@dmt0 previously it was all about the scroll being shown or not...  so not too much CSS, you can do something like this http://jsfiddle.net/handsoncode/qwwt6qyb/ but you're not able to use any external library to change a particular table.
